src/common/components/left-nav-hamburger-button.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { NamedFC } from 'common/react/named-fc';
+import { IconButton } from 'office-ui-fabric-react';
+import * as React from 'react';
+import * as styles from './left-nav-hamburger-button.scss';
+
+export type LeftNavHamburgerButtonProps = {
+    ariaLabel: string;
+};
+
+export const LeftNavHamburgerButton = NamedFC<LeftNavHamburgerButtonProps>(
+    'LeftNavHamburgerButton',
+    props => {
+        return (
+            <IconButton
+                className={styles.leftNavHamburgerButton}
+                iconProps={{ iconName: 'GlobalNavButton' }}
+                ariaLabel={props.ariaLabel}
```

Comment:
Hmmm any reason this label is coming from the parent element?

Reply:
We want different aria labels depending on whether we are in Assessment ("Assessment expand to see list of all tests and requirements") or FastPass ("FastPass expand to see a list of all tests"). Would you happen to know if there is there another place I can put/set it?